spec/system/user_uses_the_editor_spec.rb

```diff
@@ -88,7 +88,7 @@ def fill_markdown_with(content)
     end
   end
 
-  describe "using v2 editor", js: true do
+  describe "using v2 editor", js: true, stub_elasticsearch: true do
```

Comment:
This actually hit Elasticsearch once for every letter entered in the tag input 😅

Reply:
😱